test/appium/tests/atomic/chats/test_public.py

```diff
@@ -328,6 +329,304 @@ def test_public_chat_open_using_deep_link(self):
+
+        self.errors.verify_no_errors()
+
+    @marks.testrail_id(702067)
+    @marks.medium
+    def test_messaging_open_user_profile_long_press_on_message_1_1_chat(self):
+
+        message_from_sender = "Message sender"
+
+        self.home_1.just_fyi("1-1 chat: sender adds receiver and send a message")
+        self.chat_1.send_message(message_from_sender)
+        self.chat_1.chat_element_by_text(message_from_sender).long_press_element()
+        if self.chat_1.view_profile_by_avatar_button.is_element_displayed():
+            self.errors.append('Member photo is shown on long tap on sent message from 1-1 chat')
+        self.chat_1.click_system_back_button(2)
+
+        self.home_2.just_fyi("1-1 chat: receiver verifies that can open sender profile on long tap on message")
+        self.chat_2.chat_element_by_text(message_from_sender).long_press_element()
+        if self.chat_2.view_profile_by_avatar_button.is_element_displayed():
+            self.errors.append('1-1 chat: another user profile is opened on long tap on received message')
+        self.chat_2.click_system_back_button(2)
+
+    @marks.testrail_id(702068)
+    @marks.medium
+    def test_messaging_open_user_profile_long_press_on_message_in_public_chat(self):
+        message_from_receiver = "Message receiver"
+        # [chat.home_button.double_click() for chat in [self.chat_1, self.chat_2]]
+        [home.get_chat('#' + self.public_chat_name).click() for home in (self.home_1, self.home_2)]
+        self.home_1.just_fyi(
+            'Public chat: send message and verify that user profile can be opened on long press on message')
+        # chat_public_1, chat_public_2 = home_1.get_chat_view(), home_2.get_chat_view()
+        self.public_chat_2.send_message(message_from_receiver)
+        self.public_chat_2.chat_element_by_text(message_from_receiver).long_press_element()
+        if self.public_chat_2.view_profile_by_avatar_button.is_element_displayed():
+            self.errors.append('Public chat: "view profile" is shown on long tap on sent message')
+        self.public_chat_1.view_profile_long_press(message_from_receiver)
+        if not self.public_chat_1.remove_from_contacts.is_element_displayed():
+            self.errors.append('Public chat: another user profile is not opened on long tap on received message')
+
+        self.errors.verify_no_errors()
+
+        self.public_chat_1.get_back_to_home_view()
+        self.public_chat_2.click_system_back_button(2)
+
+        # [chat.home_button.double_click() for chat in [self.public_chat_1, self.public_chat_2]]
+        self.home_1.get_chat(self.default_username_2).click()
+        self.home_2.get_chat(self.default_username_1).click()
+
+    @marks.testrail_id(702069)
+    @marks.medium
+    def test_messaging_can_pin_messages_in_one_to_one_chat(self):
+        self.home_1.just_fyi("Check that Device1 can pin own message in 1-1 chat")
+        self.chat_1.send_message(self.message_1)
```

Comment:
ca add that to setup_method instead of sending 'hey'

Reply:
@churik can't find the part of code sending 'hey' message in TestMessaging group of cases. Could you please specify what do you mean.

Reply:
sorry, meant "test message".
In fact instead of sending one more message you can work with already sent (if it was sent in setup method)